Backend/src/server.ts

```diff
@@ -24,76 +23,107 @@ app.use(bodyParser.json());
 
 app.use(passport.initialize());
 
-app.use(requestLogger)
+app.use(requestLogger);
 //The regular logger needs to be before the router
-
-app.post('/auth/jwt', (req, res) => {
-    passport.authenticate(
-        'local',
-        { session: false },
-        (error: Error, user: any) => {
-            if (error || !user) {
-                res.status(400).json({ error });
-                return
-            }
-
-            const payload = {
-                useremail: user.email,
-                expires: Date.now() + 3600000,
-            };
-
-            req.login(payload, { session: false }, (error) => {
-                if (error) {
-                    res.status(400).send({ error });
-                }
-                const token = jwt.sign(JSON.stringify(payload), process.env.SECRET);
-                res.cookie('jwt', jwt, { httpOnly: true, secure: true });
-                res.status(200).send({ token: token, username: user.username, useremail: user.email });
-            });
-        },
-    )(req, res);
+app.use(requestLogger);
+
+app.post("/auth/jwt", (req, res) => {
+  passport.authenticate(
+    "local",
+    { session: false },
+    (error: Error, user: any) => {
+      if (error || !user) {
+        res.status(400).json({ error });
+        return;
+      }
+
+      const payload = {
+        useremail: user.email,
+        expires: Date.now() + 3600000,
+      };
+
+      req.login(payload, { session: false }, (error) => {
+        if (error) {
+          res.status(400).send({ error });
+        }
+        const token = jwt.sign(JSON.stringify(payload), process.env.SECRET);
+        res.cookie("jwt", jwt, { httpOnly: true, secure: true });
+        res.status(200).send({
+          token: token,
+          username: user.username,
```

Comment:
Er det din formatter eller er det dig der ændrer i måden funktioner ala den her står på? For jeg synes det gør det vanskeligt at læse så jeg sidder og retter det tilbage.